pandas/io/pytables.py

```diff
@@ -3469,14 +3469,10 @@ def write_metadata(self, key: str, values: np.ndarray):
         key : str
         values : ndarray
         """
-        # error: Incompatible types in assignment (expression has type
-        # "Series", variable has type "ndarray")
-        values = Series(values)  # type: ignore[assignment]
-        # error: Value of type variable "FrameOrSeries" of "put" of "HDFStore"
-        # cannot be "ndarray"
-        self.parent.put(  # type: ignore[type-var]
+        series = Series(values)
```

Comment:
nit, I have a preference not to name things that clash, save for the case.

i'm happy with `s` but others prefer longer names.

maybe to avoid subjective nitpicking could avoid the intermediate variable.